Tests/Marketplace/install_packs.sh

```diff
@@ -20,36 +27,37 @@ fi
 
 echo "Trying to authenticate with GCS..."
 gcloud auth activate-service-account --key-file="$GCS_MARKET_KEY" > auth.out 2>&1
-if [ $? -ne 0 ]; then
-  echo "Failed to authenticate with GCS, exiting..."
-  exit 1
-fi
+exit_on_error $? "Failed to authenticate with GCS, exiting..."
+
 echo "Successfully authenticated with GCS."
 
 echo "starting configure_and_install_packs instance role: ${INSTANCE_ROLE}"
-exit_code=0
-if [[ "${INSTANCE_ROLE}" == "XSIAM" ]]; then
 
+if [[ "${INSTANCE_ROLE}" == "XSIAM" ]]; then
   if [ -n "${CLOUD_CHOSEN_MACHINE_IDS}" ]; then
+    exit_code=0
     IFS=', ' read -r -a CLOUD_CHOSEN_MACHINE_ID_ARRAY <<< "${CLOUD_CHOSEN_MACHINE_IDS}"
     for CLOUD_CHOSEN_MACHINE_ID in "${CLOUD_CHOSEN_MACHINE_ID_ARRAY[@]}"; do
       python3 ./Tests/Marketplace/configure_and_install_packs.py -s "$SECRET_CONF_PATH" --ami_env "${INSTANCE_ROLE}" --branch "$CI_COMMIT_BRANCH" --build_number "$CI_PIPELINE_ID" --service_account $GCS_MARKET_KEY -e "$EXTRACT_FOLDER" --cloud_machine "${CLOUD_CHOSEN_MACHINE_ID}" --cloud_servers_path ${XSIAM_SERVERS_PATH} --pack_ids_to_install "$ARTIFACTS_FOLDER/content_packs_to_install.txt" --cloud_servers_api_keys $XSIAM_API_KEYS
       if [ $? -ne 0 ]; then
         exit_code=1
+        echo "Failed to install packs on machine ${CLOUD_CHOSEN_MACHINE_ID}"
       fi
     done
-    echo "Finished configure_and_install_packs script with exit code ${exit_code}"
-    exit "${exit_code}"
+
+    exit_on_error "${exit_code}" "Finished configure_and_install_packs script with exit code ${exit_code}"
```

Comment:
Is it an error? it'll print "ERROR: " in front of the message.
Also:
```suggestion
    exit_on_error "${exit_code}" "Finished configure_and_install_packs script"
```